components/script/dom/webglframebuffer.rs

```diff
@@ -251,7 +251,8 @@ impl WebGLFramebuffer {
         Ok(())
     }
 
-    pub fn detach_renderbuffer(&self, rb: &WebGLRenderbuffer) {
+    fn call_on_matching_renderbuffers<F>(&self, rb: &WebGLRenderbuffer, closure: F)
+        where F: Fn(&DOMRefCell<Option<WebGLFramebufferAttachment>>) {
```

Comment:
Also, you can probably use `FnMut` instead of `Fn`, which would allow you to use closures that mutate other state.